bin/spack

```diff
@@ -11,7 +11,7 @@
 # See https://stackoverflow.com/a/47886254
 """:"
 # prefer python3, then python, then python2
-for cmd in python3 python python2; do
+for cmd in python3.7 python python2; do
```

Comment:
Why this? I don't think I'm OK with that.

Reply:
That’s my local change got pushed, I am still debugging this! Will cleanup once ready.